.github/workflows/r_package.yml

```diff
@@ -3,11 +3,11 @@ name: R-package
 on:
   push:
     branches:
-    - master
+      - master
```

Comment:
can you please revert all these unrelated formatting changes?

Reply:
I really have to disable my format-on-save here 😬